CHANGES.md

```diff
@@ -1,5 +1,6 @@
 # Future
 - LINUX: Added support for Linux GameMode (https://github.com/FeralInteractive/gamemode), which can be toggled on/off in the Power Management or Latency settings menus.
+- Added support for showing the overlay behind the menu instead of in front. This is currently only supported on the GL and Vulkan drivers.
```

Comment:
```suggestion
- Added support for showing the overlay behind the menu instead of in front. This is currently only supported on the GL, Vulkan and 3DS drivers.
 ```